src/vcpkg/base/strings.cpp

```diff
@@ -315,17 +315,24 @@ std::vector<std::string> Strings::split(StringView s, const char delimiter)
     }
 }
 
-std::vector<std::string> Strings::split_keep_empty(StringView s, const char delimiter)
+std::vector<std::string> Strings::split_lines_keep_empty(StringView s)
 {
     std::vector<std::string> output;
     auto first = s.begin();
     const auto last = s.end();
     do
     {
-        auto next = std::find_if(first, last, [=](const char c) { return c == delimiter; });
+        auto next = std::find_if(first, last, [=](const char c) { return c == '\r' || c == '\n'; });
         output.emplace_back(first, next);
         if (next == last) return output;
-        first = next + 1;
+        if (next + 1 != last && next[0] == '\r' && next[1] == '\n')
+        {
+            first = next + 2;
+        }
+        else
+        {
+            first = next + 1;
+        }
```

Comment:
```suggestion
        first = next + 1;
        if (first != last && *next == '\r' && *first == '\n') { ++first; }
```

I think this more clearly says "we consumed the \r, then if we also see \n there we consume the \n"